apps/frontend/src/libs/components/link/link.tsx

```diff
@@ -7,7 +7,7 @@ import styles from "./styles.module.css";
 
 type Properties = {
 	children: React.ReactNode;
-	to: ValueOf<typeof AppRoute>;
+	to: "#" | ValueOf<typeof AppRoute>;
```

Comment:
We don't need this here for now
```suggestion
	to: ValueOf<typeof AppRoute>;
```